src/main/java/org/snomed/snowstorm/core/data/domain/Relationship.java

```diff
@@ -187,30 +187,28 @@ public boolean isGrouped () {
 	}
 
 	public ConcreteValue getConcreteValue() {
-		if (this.value == null) {
-			return null;
-		}
-
-		this.concreteValue = new ConcreteValue(this.value);
 		return this.concreteValue;
 	}
 
 	public void setConcreteValue(ConcreteValue concreteValue) {
 		this.concreteValue = concreteValue;
 	}
 
-	public void setConcreteValue(String value) {
-		if (value != null) {
-			this.concreteValue = new ConcreteValue(value);
+	public void setConcreteValueFromExternal(org.snomed.otf.owltoolkit.domain.Relationship.ConcreteValue concreteValue) {
```

Comment:
I changed the name of the method to setConcreteValueFromExternal as there would have been two methods named setConcreteValue that had a parameter of ConcreteValue.

Reply:
This method call has now been reverted back to setConcreteValue with two String arguments.